db/migrate/20170106175305_rename_minter_state_random_to_rand.active_fedora_noid_engine.rb

```diff
@@ -0,0 +1,7 @@
+# This migration comes from active_fedora_noid_engine (originally 20161021203429)
+# frozen_string_literal: true
+class RenameMinterStateRandomToRand < ActiveRecord::Migration
+  def change
+    rename_column :minter_states, :random, :rand
```

Comment:
Do you want to merge this migration with the other one?  That's few migrations to track.